### ISSUE TITLE
Strange work of Charcoal Pile Igniter

### ISSUE THREAD

**Issue by MadRatSRP** [GTNewHorizons/GT-New-Horizons-Modpack#10298]
### Your GTNH Discord Username

MadRat96#9831

### Your Pack Version

2.1.2.1

### Your Server

SP

### Type of Server

Single Player

### Your Expectation

Charcoal Pile Ingiter works strangely, i did these steps to get charcoal 2 times:
1)Loaded wood blocks into multiblock, closed it with dirt
2)Igniter showed the animation and started working
3)Gone to sleep (Somnia installed)
4)Woke up, came back to multiblock, checked blocks (enough time passed for it to turn blocks into coal) - still wood blocks instead of coal blocks
5)Turned Igniter off with mallet - right after i did it blocks turned into charcoal blocks
6)Loaded wood blocks again, came back to igniter (which was off), clicked on it with mallet - right after it blocks turned again into charcoal blocks.

So i suggest fixing 2 things with it:
1)After Igniter done with its job, do not force player to click on it with mallet to turn wood blocks into coal - it should be done automatically ater job is done
2)Check is Igniter off or on, if its off it shouldnt create a bunch of coal after turning it on like a wizard and should work properly :D

### The Reality

null

### Your Proposal

null

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
seems to work 100% fine in .3qf for me. Also dont need a soft mallet.

are you maybe not chunkloading it?

More importantly: do you have any instruction that might help to try to reproduce your problems? Otherwise this issue is unlikely to lead anywhere.

**Comment by MadRatSRP** [comment]
> seems to work 100% fine in .3qf for me. Also dont need a soft mallet.
> 
> are you maybe not chunkloading it?
> 
> More importantly: do you have any instruction that might help to try to reproduce your problems? Otherwise this issue is unlikely to lead anywhere.

You're right, i forgot to chunkload it xD I'll re-test again after chunkloading
